### PR TITLE
Capstone completion

### PR DESCRIPTION
**Pull request by wayungi**
This PR adds -1: 
- Mobile version of the capstone project
- The large screen version of the capstone project
- Dynamic generation of the speakers' section

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+      </nav>
+      <p class="tagline title-color pad large-margin">"Nyegenyege Festival"</p>
+      <h1 class="main-title title-color pad large-margin mt1">THE MTN NYEGENYEGE FESTIVAL 2022</h1>
+      <p class="brief-info large-margin">Nyege Nyege is a yearly 4-day music festival held in the first 
+        week of September in Jinja, Uganda showcasing developments in music across 
+        the African continent.
+      </p>
+      <p class="event-date pad large-margin">2022.04.25(MON) ~ 29(FRI)<p>
+      <p class="venue pad large-margin">@ The Nile Hotel in Jinja</p>
+      </div>
```
Comment (VICTOR-CHUKWU) [inline index.html:54]: - [ ] Nice work @wayungi , kindly stick to the design as the section with the main title looks bigger than what you currently have in your design.
Project design :point_down: 
image
your design :point_down: 
image

- [ ] Also your background image is not covering the full width of the section screen, kindly fix

File: index.html [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+    <section class="flex-column column-center large-screen">
+      <div class="partners-large column-center flex-column">
+        <h2 class="sub-title-2 ">Partner</h2>
+        <span class="red-bar bar"></span>
+        <div class="logos flex-row wrap row-center">
+          <p class="mtn">MTN</p>
+          <p class="silk">silkEvents</p>
+          <p class="guvnor">guvnor</p>
+          <p class="jonam">JONAM</p>
+          <p class="game">Game Hub</p>
```
Comment (VICTOR-CHUKWU) [inline index.html:116]: - [ ] Kindly align the patners element to the center as it currently sits close to the bottom of the section

**Review by VICTOR-CHUKWU — CHANGES_REQUESTED** [review]
Hi @wayungi  

Good job so far!
:heavy_check_mark: Good Pr
:heavy_check_mark: all Linters passed

There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: index.html [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+        </div>
+      </div>
+      <button type="submit" class="join-nyege small-screen">Join Nyegenyege festival 2022</button>
+      <a href="#" class="whole-program large-screen">SEE THE WHOLE PROGRAM</a>
+    </section>
+
+    <!-- featured speakers section -->
+    <section class="feat-speakers flex-column column-center">
+      <h2 class="sub-title">Main program</h2>
+      <span class="red-bar"></span>
```
Comment (micheaol) [inline index.html:104]: ## [OPTIONAL]:

Please ensure that the pages look almost identical to the original design.

Your Implementation | Design
:---:|:---:
image | image

## [OPTIONAL]:

Your Implementation | Figma Design
:---:|:---:
image | image

**Review by micheaol — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 
Hi @wayungi , 

Great work on making the changes 👏 
In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 

well-1
Good luck! 🎉 


Cheers and Happy coding!👏👏👏

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**